src/test/java/homework4/CollectionsIntoMapTest.java

```diff
@@ -0,0 +1,45 @@
+package homework4;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Map;
+
+import static homework4.CollectionsIntoMap.convertCollectionsIntoMap;
+import static homework4.CollectionsIntoMap.convertCollectionsIntoMapException;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class CollectionsIntoMapTest {
```

Comment:
Please reformat this class. Long assertions can be hard to read (you can try the reformat code action, if it doesn't work - do it manually):
<img width="1633" alt="image" src="https://user-images.githubusercontent.com/90613721/182466261-d9f2fc65-6161-49e6-940b-8f0859f506d8.png">

Reply:
Thanks, done